pages/about.js

```diff
@@ -19,26 +22,53 @@ export default function About(props) {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <section className="layout-container relative mb-28">
```

Comment:
Looks awesome! I think we could reduce the bottom margin here a fair bit. 
![image](https://user-images.githubusercontent.com/31868510/122073959-d377e200-cdc6-11eb-80a2-7d3a28077053.png)

Reply:
Reduced the space by a little over half of what's seen here.